Ray.jl/test/function_manager.jl

```diff
@@ -48,6 +48,24 @@
     fc2 = import_function!(fm, fcd, jobid)
     @test fc.(1:10) == fc2.(1:10) != f.(1:10)
 
+    @testset "warn/error for large functions" begin
+        # for some reason, using `let` to introduce local scope does not work
+        # during test execution to generate a closure, even though it works
+        # locally.  so we use a factory intsead:
+        function bigfactory(size)
+            x = rand(UInt8, size)
+            return f(y) = y * sum(x)
```

Comment:
might be better to locally define a new `f` since it's not obvious when it's declared that another unit test depends on it?

Reply:
it ends up being an anonymous function anyway (due to the local scope)